src/components/BodySchema/BodySchema.js

```diff
@@ -18,45 +18,108 @@ export default class BodySchema extends Component {
   }
 
   render () {
-    const { properties, styleVariation, classes, hasTabs } = this.props
+    const { properties, additionalProperties, styleVariation, classes, hasTabs } = this.props
 
     if (!properties) {
       return null
     }
+
     const { expandedProp } = this.state
+    const hasAdditionalProperties = properties.find(
```

Comment:
This variable name (`hasAdditionalProperties`) is confusing. It is an object, but the name indicates it's a boolean.

Perhaps`foundAdditionalProperties` or sth indicating a noun.